### PR TITLE
ghq 1.0.0

### PR DESCRIPTION
**Pull request by f440**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by SMillerDev** [comment]
I think it got retagged. Can you check with upstream?

**Comment by f440** [comment]
I corrected the wrong checksum. Thank you for your pointed out.

File: Formula/ghq.rb [hunk 1]
```diff
@@ -2,8 +2,8 @@ class Ghq < Formula
   desc "Remote repository management made easy"
   homepage "https://github.com/motemen/ghq"
   url "https://github.com/motemen/ghq.git",
-      :tag      => "v0.99.2",
-      :revision => "69aea14edeb7b7b213528445c9dc8c0e082bb171"
+      :tag      => "v1.0.0",
```
Comment (chenrui333) [inline Formula/ghq.rb]: maybe bump to the new release 1.0.1?

Comment (chenrui333) [reply Formula/ghq.rb]: I will do a in separate PR, should be trivial though. :)

**Review by chenrui333 — APPROVED** [review]

**Comment by chenrui333** [comment]
Thanks @f440!!

File: Formula/ghq.rb (hunk 1 above)
Comment (chenrui333) [reply Formula/ghq.rb]: https://github.com/Homebrew/homebrew-core/pull/48543

**Pull request closed without merging**